x-pack/plugins/log_explorer/public/components/common/popover_chip.tsx

```diff
@@ -78,15 +78,12 @@ export function ChipWithPopover({
         font-size: ${xsFontSize};
         display: flex;
         justify-content: center;
+        margin-top: -3px;
```

Comment:
Fix required to centre the Popover chip inside a cell as Table padding pushes the Chip towards the bottom